src/tarantool-gdb.py

```diff
@@ -1399,3 +1415,540 @@ def invoke(self, arg, from_tty):
+            ('swim::dissemination_queue', 'swim_member::in_dissemination_queue'),
+            ('swim::on_member_event', 'trigger::link'),
+            ('swim::round_queue', 'swim_member::in_round_queue'),
+            ('swim_scheduler::queue_output', 'swim_task::in_queue_output'),
+            ('tx_manager::read_view_txs', 'txn::in_read_view_txs'),
+            ('tx_manager::all_stories', 'memtx_story::in_all_stories'),
+            ('tx_manager::traverse_all_stories', 'memtx_story::in_all_stories'), # struct rlist*
+            ('tx_manager::all_txs', 'txn::in_all_txs'),
+            ('txn::conflict_list', 'tx_conflict_tracker::in_conflict_list'),
+            ('txn::conflicted_by_list', 'tx_conflict_tracker::in_conflicted_by_list'),
+            ('txn::gap_list', 'gap_item::in_gap_list'),
+            ('txn::full_scan_list', 'full_scan_item::in_full_scan_list'),
+            ('txn::on_commit', 'trigger::link'),
+            ('txn::on_rollback', 'trigger::link'),
+            ('txn::on_wal_write', 'trigger::link'),
+            ('txn::point_holes_list', 'point_hole_item::in_point_holes_list'),
+            ('txn::read_set', 'tx_read_tracker::in_read_set'),
+            ('txn::savepoints', 'txn_savepoint::link'),
+            ('txn_limbo::queue', 'txn_limbo_entry::in_queue'),
+            ('txn_stmt::on_commit', 'trigger::link'),
+            ('txn_stmt::on_rollback', 'trigger::link'),
+            ('sql_stmt_cache::gc_queue', 'stmt_cache_entry::link'),
+            ('sys_alloc::allocations', 'container::rlist'),
+            ('user::credentials_list', 'credentials::in_user'),
+            ('vy_cache_env::cache_lru', 'vy_cache_node::in_lru'),
+            ('vy_history::stmts', 'vy_history_node::link'),
+            ('vy_join_ctx::entries', 'vy_join_entry::in_ctx'),
+            ('vy_lsm::on_destroy', 'trigger::link'),
+            ('vy_lsm::runs', 'vy_run::in_lsm'),
+            ('vy_lsm::sealed', 'vy_mem::in_sealed'),
+            ('vy_lsm_recovery_info::ranges', 'vy_range_recovery_info::in_lsm'),
+            ('vy_lsm_recovery_info::runs', 'vy_run_recovery_info::in_lsm'),
+            ('vy_quota::wait_queue', 'vy_quota_wait_node::in_wait_queue'),
+            ('vy_range::slices', 'vy_slice::::in_range'),
+            ('vy_range_recovery_info::slices', 'vy_slice_recovery_info::in_range'),
+            ('vy_recovery::lsms', 'vy_lsm_recovery_info::in_recovery'),
+            ('vy_tx::on_destroy', 'trigger::link'),
+            ('vy_tx_manager::read_views', 'vy_read_view::in_read_views'),
+            ('vy_tx_manager::writers', 'vy_tx::in_writers'),
+            ('vy_write_iterator::src_list', 'vy_write_src::in_src_list'),
+            ('wal_writer::watchers', 'wal_watcher::next'),
+            ('watchable::pending_watchers', 'watcher::in_idle_or_pending'),
+            ('watchable_node::all_watchers', 'watcher::in_all_watchers'),
+            ('watchable_node::idle_watchers', 'watcher::in_idle_or_pending'),
```

Comment:
I'd like to remind that initially it was constant and I put it inline to tiе tuple composition and decomposition as close as possible instead of introducing named tuple. For that reason I would keep it inline.